internal/charmstore/search.go

```diff
@@ -614,8 +647,19 @@ func createSearchDSL(sp SearchParams) elasticsearch.QueryDSL {
 // that key. The created filter will only match when at least one of the
 // requested values matches for all of the requested keys. Any filter names
 // that are not defined in the filters map will be silently skipped.
-func createFilters(f map[string][]string, admin bool, groups []string) elasticsearch.Filter {
-	af := make(elasticsearch.AndFilter, 0, len(f)+1)
+func createFilters(f map[string][]string, admin bool, groups []string, expandedMultiSeries bool) elasticsearch.Filter {
```

Comment:
There are quite a number of arguments here, and they're all represented by
fields in SearchParams, so perhaps it might make sense to pass SearchParams
in as an argument?